hivemind/utils/grpc.py

```diff
@@ -208,3 +223,28 @@ def combine_from_streaming(stream: Iterable[runtime_pb2.Tensor]) -> runtime_pb2.
         buffer_chunks.append(tensor_part.buffer)
     serialized_tensor.buffer = b"".join(buffer_chunks)
     return serialized_tensor
+
+
+StreamMessage = TypeVar("StreamMessage")
+
+
+async def gather_from_streaming(
```

Comment:
In a future PR, we should remove `grpc.py` and move this method to a new file (e.g., `hivemind/utils/streaming.py`). But we can keep it as is in this PR :)

Reply:
We can do this particular change right now. It seems very easy to do